plugin.video.eventvods/addon.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.eventvods" name="Eventvods" version="1.0.1+matrix.1" provider-name="Ikosse">
+    <requires>
+        <import addon="xbmc.python" 			version="3.0.0" />
+        <import addon="script.module.requests"	version="2.22.0+matrix.1"/>
+        <import addon="plugin.video.tubed"		version="1.0.3+matrix.1" optional="true"/>
+        <import addon="plugin.video.youtube" 	version="6.8.10+matrix.1"/>
+        <import addon="plugin.video.twitch" 	version="2.5.3+matrix.1"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">Catch up on esports events completely spoiler free.</summary>
+        <summary lang="sv_SE">Få tillgång till de senaste esports-evenemangen helt utan spoilers.</summary>
+    	<description lang="en_GB">This is an unofficial addon that gives you access to the latests esports events from eventvods.com.</description>
+    	<description lang="sv_SE">Detta är ett inofficielt addon som ger dig tillgång till de senaste esports-evenemangen från eventvods.com.</description>
+        <license>GNU General Public License, v2</license>
```

Comment:
GPL-2.0-or-later or GPL-2.0-only (spdx identifier)

Reply:
Thank you for the feedback! I have modified the addon.xml and followed the instructions on https://kodi.wiki/view/HOW-TO:Update_a_pull_request_to_an_official_addon_repository_on_GitHub, I think. I'm still quite novice on git; is there something more I should do?

Reply:
All good thanks